web-console/knikubevirt/openshift-vms-overview/openshift-vms-overview.md

```diff
@@ -0,0 +1,25 @@
+# OpenShift VMs Overview
+
+The purpose of this PR is to explore what Virtual Machines could look like in the Overview and Resources sections of Openshift/CNV.
+This is the first pass so really just trying to understand exactly what needs to be surfaced there.
+
+Through the Overview page the user can see the VM’s based on the application it is associated with via it’s label and and also see some sparkline charts that illustrate the usage of Network, CPU and Memory for the past 7 days.
+
+![sparkline](Openshift-overview-virtual-machine-sparkline.png)
```

Comment:
Last nitpick from me! Going to approve knowing this one item will be updated :) Would you mind shifting the metrics and number of pods to the right? They are right aligned in the current implementation.